xalienfs.sh

```diff
@@ -36,3 +42,26 @@ export CXXFLAGS="$CXXFLAGS -I$XROOTD_ROOT/include -I$XROOTD_ROOT/include/xrootd/
 make
 make install INSTALLSITEARCH=$INSTALLROOT/lib/perl \
              INSTALLARCHLIB=$INSTALLROOT/lib/perl
+
+# Modulefile
+MODULEDIR="$INSTALLROOT/etc/modulefiles"
+MODULEFILE="$MODULEDIR/$PKGNAME"
+mkdir -p "$MODULEDIR"
+cat > "$MODULEFILE" <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0 XRootD/${XROOTD_VERSION}-${XROOTD_REVISION}             \\
+            ${OPENSSL_VERSION:+OpenSSL/$OPENSSL_VERSION-$OPENSSL_REVISION}   \\
+            AliEn-Runtime/${ALIEN_RUNTIME_VERSION}-${ALIEN_RUNTIME_REVISION}
+
+# Our environment
+setenv XALIENFS_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+prepend-path LD_LIBRARY_PATH \$::env(XALIENFS_ROOT)/lib
```

Comment:
Do you really need the environment variable? can't you simply use a local variable?

Reply:
I tried, it didn't work. I will try again and ask for help if it still doesn't work.